Tests/CubeService_test.py

```diff
@@ -135,6 +137,12 @@ def test_get_all_names(self):
 
         self.tm1.cubes.delete(cube_name)
 
+        cube = self.tm1.cubes.get_control_cubes()[0]
```

Comment:
tests should be non-invasive in a way that they only manipulate objects that they created themself and not interact with the rest of the model. 

Please create a dummy control cube instead

Reply:
That's good feedback, I'll update.

Reply:
updated with https://github.com/cubewise-code/tm1py/pull/723/commits/3ae357020d0793f6bbdf595af7137860c3169ae9